docs/src/main/paradox/play-framework.md

```diff
@@ -50,3 +54,58 @@ Scala
 Java
 :   @@snip[MyController.java]($root$/../play-interop-test-java/src/main/java/controllers/MyController.java) { #using-client }
 
+## Serving gRPC from a Play Framework app
+
+To be able to serve gRPC from a Play Framework app you must enable [HTTP/2 Support](https://www.playframework.com/documentation/2.6.x/AkkaHttpServer#HTTP%2F2-support-%28experimental%29)
+with HTTPS and the ALPN agent. (This is still somewhat involved and we hope to simplify it)
+
+Generating classes from the gRPC service definition is done buy adding the Akka gRPC plugin to your sbt build:
+
+sbt
+:   @@@vars
+    ```scala
+    // in project/plugins.sbt:
+    addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "$projectversion$")
+    ```
+    @@@
+
+
+Then you need to enable the Play server side code generator in `build.sbt`:
+
+Scala
```

Comment:
This should eventually be sbt/maven etc, not by lang

Reply:
No, because Play with sbt is the default and maven is semi-supported, and these are different configurations to trigger generation of Scala or Java client/interfaces for the service.

Reply:
However, I see now that I made a copy-pasta-miss here: importing the server and using the client generator. D'oh!

Reply:
Fixed in #304

Reply:
(tho here is no maven here, i know, so just sbt)